libs/akita-ng-router-store/src/lib/router.spec.ts

```diff
@@ -0,0 +1,130 @@
+import { APP_BASE_HREF } from '@angular/common';
+import { Component, Injectable, NgZone } from '@angular/core';
+import { fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { ActivatedRouteSnapshot, CanActivate, Router, Routes } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AkitaNgRouterStoreModule } from '..';
+import { RouterQuery } from './router.query';
+import { RouterStore } from './router.store';
+
+@Component({
+  selector: 'test-empty',
+  template: '',
+})
+class EmptyComponent {}
+
+const intentionalTestError = new Error('Intentional test error.');
```

Comment:
Thanks, great specs. Can we add one simple spec for a case where we don't use `canActivate`, please?

Reply:
Added!